### PR TITLE
Support for Meraki Z series appliances

### PR DESCRIPTION
**Pull request by corsoblaster**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [ ] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

#### Testers

If you would like to test this pull request then please run: `./scripts/github-apply <pr_id>`, i.e `./scripts/github-apply 5926`
After you are done testing, you can remove the changes with `./scripts/github-remove`.  If there are schema changes, you can ask on discord how to revert.

### CODE REVIEW

**Comment by PipoCanaja** [comment]
Hi @corsoblaster 
Thanx for your PR. Looks good so far. 
Could you add test data to it please?  That will make Travis tests happy :)
Bye

File: includes/polling/os/merakiz.inc.php [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ *
+ * Copyright (c) 2015 Will Jones <email@willjones.eu>
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+
+if (empty($hardware)) {
```
Comment (laf) [inline includes/polling/os/merakiz.inc.php]: No point in doing that check.

**Comment by laf** [comment]
@corsoblaster Also still needs tests if you can get that done please. https://docs.librenms.org/Developing/os/Test-Units/

**Comment by corsoblaster** [comment]
Hi,

sorry but I have no Meraki Z at the moment in my hand to complete the test.

 

I’ll as soon as possible.

 

 

Da: Neil Lathwood <notifications@github.com>
Risposta: librenms/librenms <reply@reply.github.com>
Data: martedì 20 agosto 2019 15:17
A: librenms/librenms <librenms@noreply.github.com>
Cc: "a.corsini2@gmail.com" <a.corsini2@gmail.com>, Mention <mention@noreply.github.com>
Oggetto: Re: [librenms/librenms] Support for Meraki Z series appliances (#10264)

 

@corsoblaster Also still needs tests if you can get that done please. https://docs.librenms.org/Developing/os/Test-Units/

**Comment by laf** [comment]
@corsoblaster Any news on this?

**Comment by corsoblaster** [comment]
Yes. Device on my hands.

I’m preparing environment installation to make tests

 

 

Da: Neil Lathwood <notifications@github.com>
Risposta: librenms/librenms <reply@reply.github.com>
Data: martedì 22 ottobre 2019 02:11
A: librenms/librenms <librenms@noreply.github.com>
Cc: "a.corsini2@gmail.com" <a.corsini2@gmail.com>, Mention <mention@noreply.github.com>
Oggetto: Re: [librenms/librenms] Support for Meraki Z series appliances (#10264)

 

@corsoblaster Any news on this?

**Comment by Jellyfrog** [comment]
@corsoblaster any news?

**Comment by corsoblaster** [comment]
Hi,

I’ll complete the tests during the Season holidays 😊

 

 

Da: Jellyfrog <notifications@github.com>
Risposta: librenms/librenms <reply@reply.github.com>
Data: martedì 17 dicembre 2019 17:59
A: librenms/librenms <librenms@noreply.github.com>
Cc: "a.corsini2@gmail.com" <a.corsini2@gmail.com>, Mention <mention@noreply.github.com>
Oggetto: Re: [librenms/librenms] Support for Meraki Z series appliances (#10264)

 

@corsoblaster any news?

**Comment by Jellyfrog** [comment]
@corsoblaster Closing this due to inactivity, please feel free to reopen if you continue working on it!

**Pull request closed without merging**